lib/compat/wordpress-6.0/block-patterns-update.php

```diff
@@ -151,6 +151,306 @@ function gutenberg_register_gutenberg_patterns() {
+
+			<!-- wp:post-date {"fontSize":"extra-small"} /-->
+
+			<!-- wp:spacer {"height":"20px"} -->
+			<div style="height:20px" aria-hidden="true" class="wp-block-spacer"></div>
+			<!-- /wp:spacer -->
+
+			<!-- wp:separator {"opacity":"css","className":"is-style-wide"} -->
+			<hr class="wp-block-separator has-css-opacity is-style-wide"/>
+			<!-- /wp:separator -->
+
+			<!-- wp:spacer {"height":"20px"} -->
+			<div style="height:20px" aria-hidden="true" class="wp-block-spacer"></div>
+			<!-- /wp:spacer -->
+			<!-- /wp:post-template --></div>
+			<!-- /wp:query --></div>
+			<!-- /wp:column --></div>
+			<!-- /wp:columns -->',
+		),
+		'query-more-posts'                   => array(
+			'title'      => _x( 'Featured post with more posts', 'Block pattern title', 'gutenberg' ),
+			'blockTypes' => array( 'core/query' ),
+			'categories' => array( 'query' ),
+			'content'    => '<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|30","right":"var:preset|spacing|30","bottom":"var:preset|spacing|30","left":"var:preset|spacing|30"}}},"backgroundColor":"black"} -->
+			<div class="wp-block-group alignwide has-black-background-color has-background" style="padding-top:var(--wp--preset--spacing--30);padding-right:var(--wp--preset--spacing--30);padding-bottom:var(--wp--preset--spacing--30);padding-left:var(--wp--preset--spacing--30)"><!-- wp:query {"query":{"perPage":1,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":false},"displayLayout":{"type":"list"}} -->
+			<div class="wp-block-query"><!-- wp:post-template -->
+			<!-- wp:columns {"verticalAlignment":"center","textColor":"white"} -->
+			<div class="wp-block-columns are-vertically-aligned-center has-white-color has-text-color"><!-- wp:column {"verticalAlignment":"center","width":"20%"} -->
+			<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:20%"><!-- wp:post-author {"showAvatar":false} /--></div>
+			<!-- /wp:column -->
+
+			<!-- wp:column {"verticalAlignment":"center","width":"66.66%"} -->
+			<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:66.66%"><!-- wp:post-date /--></div>
+			<!-- /wp:column --></div>
+			<!-- /wp:columns -->
+
+			<!-- wp:post-featured-image {"isLink":true,"align":"wide"} /-->
+
+			<!-- wp:post-title {"isLink":true,"style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}}}} /-->
+
+			<!-- wp:post-excerpt {"style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}}},"textColor":"white"} /-->
+
+			<!-- wp:spacer {"height":"25px"} -->
+			<div style="height:25px" aria-hidden="true" class="wp-block-spacer"></div>
+			<!-- /wp:spacer -->
+			<!-- /wp:post-template --></div>
+			<!-- /wp:query -->
+
+			<!-- wp:paragraph {"textColor":"white"} -->
+			<p class="has-white-color has-text-color"><strong>More Posts</strong></p>
```

Comment:
```suggestion
			<p class="has-white-color has-text-color"><strong>' . _x( 'More Posts', 'block pattern sample content', 'gutenberg' ) . '</strong></p>
```

Adds the translation function.

Reply:
Thank you!